### ISSUE TITLE
Unable to select customPropertiesFields

### ISSUE THREAD

**Issue by dclawson** [ebess/advanced-nova-media-library#74]
I have added Alt and Caption as a `customPropertiesFields` array:

```
return [
            ID::make()->sortable(),
            Images::make('Logo', 'logo')
                ->thumbnail('thumb')
                ->customPropertiesFields([
                    Text::make('Alt Text'),
                    Text::make('Caption'),
                ])
                ->hideFromIndex(),
            SluggableText::make('Name')->sortable(),
            Slug::make('Slug')->hideFromIndex(),
            Text::make('Website URL', 'url')->hideFromIndex()
]
```

When clicking on the icon nothing happens, I expect the popup to display.

There are no console errors and the crop tool works perfectly. 

Could I have missed something?

**Comment by mpanius** [comment]
Same for me, popup not displaying

**Comment by ebess** [comment]
@mpanius @dclawson released 2.5.2 fix, ty!
